modules/contact/tutorials/introduction/plot02.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
We don't need to test this, right?

Reply:
@loganharbour do we test all python files?

Reply:
Nope. You'd need to add a test spec

Reply:
do we do that for most other python files though?

Reply:
Only in the framework.

Reply:
no, it's a good idea to test this, but I wasn't planning on checking in the data. I probably should though. And along with it I should add tests for _all_ tutorial examples!

Reply:
> And along with it I should add tests for _all_ tutorial examples!

![image](https://user-images.githubusercontent.com/22062133/135528479-359e3ccc-58f4-441e-ba0b-7ff3b700a308.png)

Reply:
Well, yeah, I obvoiusly meany _all_ of _my_ tutorial files...